src/App.tsx

```diff
@@ -1,15 +1,16 @@
-import React from "react";
+import CssBaseline from "@mui/material/CssBaseline";
 
 import { useState } from "react";
 import reactLogo from "./assets/react.svg";
 import viteLogo from "/vite.svg";
 import "./App.css";
+import { Button } from "@mui/material";
```

Comment:
should not import components from mui in any file
create a button component under components folder and import button from components folder